bitnami/kafka/Chart.yaml

```diff
@@ -42,4 +42,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 28.3.0
+version: 28.3.1
```

Comment:
This change breaks backward compatibility, so this must be a major release and therefore be documented in the README.md.

As an alternative, I would suggest keeping the value `extraConfig` as it is, and instead add this functionality in a new value `extraConfigYaml`. So we would have two methods to extend the configuration: as plaintext and as a yaml object.

Because kafka settings usually include dots, I would understand not all users may like the yaml approach which would require quoting the object keys.

Reply:
Additionally, the current `extraConfig` is rendered as a template, which may allow other use-cases that YAML format may not.

Reply:
I've changed it and followed your extraConfigYaml proposal.